src/components/Icon/components/IconActionEmptyCircleSplit.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const IconActionEmptyCircleSplit = ({
```

Comment:
Minor: For naming precedence would it make more sense to have `IconActionCircleSplitEmpty` - _type_ then _variant_ then _state_